__init__.py

```diff
@@ -125,6 +125,7 @@ def update_librespot():
         print('Librespot Update failed, {}'.format(repr(e)))
 
 
+@intent_handler(['WhatSong.intent', 'WhatAlbum.intent', 'WhatArtist.intent'])
```

Comment:
Does this work?

This function has no access to the bus and when called the status will be a Message and not a status dict.

Reply:
Good point, thanks. I think I originally thought this would be required for the intents, but looking at it again I think it's actually doing nothing. I've removed it and it seems to have had no effect; I'll do a few more tests later to double check, but I think this can be removed.